src/app/icd/IcdManagementServer.h

```diff
@@ -66,9 +66,9 @@ class IcdManagementServer
     IcdManagementServer() = default;
 
     static IcdManagementServer mInstance;
-    uint32_t mIdleInterval           = CHIP_CONFIG_ICD_IDLE_MODE_INTERVAL;
-    uint32_t mActiveInterval         = CHIP_CONFIG_ICD_ACTIVE_MODE_INTERVAL;
-    uint16_t mActiveThreshold        = CHIP_CONFIG_ICD_ACTIVE_MODE_THRESHOLD;
+    uint32_t mIdleInterval           = CHIP_CONFIG_ICD_IDLE_MODE_INTERVAL;    // in seconds
```

Comment:
can you add a static assert that `CHIP_CONFIG_ICD_IDLE_MODE_INTERVAL <= 64800`

Reply:
done